examples/gzip/Dockerfile-service

```diff
@@ -0,0 +1,13 @@
+FROM debian:buster-slim
+
+RUN apt-get update \
+    && apt-get install --no-install-recommends -y python3 python3-pip \
+    && apt-get autoremove -y && apt-get clean \
+    && rm -rf /tmp/* /var/tmp/* \
+    && rm -rf /var/lib/apt/lists/*
+RUN pip3 install -q flask
+RUN mkdir -p /code/data
+RUN dd if=/dev/zero of="/code/data/file.txt" bs=1024 count=102400 && \
```

Comment:
lets make it 10M rather than 100M (apologies - i know this was my suggestion)

also - a stylistic point - please place `&&` and beginning of following line - it makes code more readable